AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -639,6 +639,7 @@ reddit.com#@#.ADBAR
 ! SECTION: Anti-adblock - Regular rules
 ! Do not forget about '!#safari_cb_affinity' section for exclusions
 !
+gohatori.com#%#//scriptlet("abort-current-inline-script", "setTimeout", "none")
```

Comment:
Better with `'` instead of `"`, like in our doc.
Also this scriptlet is not working on iOS. It this case `gohatori.com#@##adframe:not(frameset)` is required(Content Blocker for Android also does not support advanced rules).

Reply:
Done.